packages/cfd/src/Components/props.types.ts

```diff
@@ -8,6 +8,8 @@ export type TCFDPlatform = 'dxtrade' | 'mt5' | 'ctrader';
 
 export type TCFDsPlatformType = 'dxtrade' | 'mt5' | 'ctrader' | '';
 
+export type TProducts = 'swap_free' | 'zero_spread' | 'ctrader' | 'derivx' | 'cTrader' | '';
```

Comment:
What condition does the empty string covers?

Reply:
We're receiving empty string for all of our products except `swap_free` and `zero_spread`. Made changes based on your comment